app/src/main/java/com/yeonae/chamelezone/view/search/KeywordTabFragment.kt

```diff
@@ -26,6 +26,21 @@ class KeywordTabFragment : Fragment() {
         )
     )
     private val searchRvAdapter = SearchRvAdapter(searchList)
+
+    private var listener: OnKeywordSelectedListener? = null
```

Comment:
```suggestion
    private lateinit var listener: OnKeywordSelectedListener
```
null 이 필요한 상황이 아니라면, 사용하지 않는것이 좋을 것 같아요

Reply:
수정했습니다